clients/da-vinci-client/src/main/java/com/linkedin/davinci/validation/PartitionTracker.java

```diff
@@ -567,17 +566,15 @@ private void validateSequenceNumber(
         logger.error(
             "Encountered missing data message within the log compaction time window. Error msg: {}",
             dataMissingException.getMessage());
-        if (errorMetricCallback.isPresent()) {
-          errorMetricCallback.get().execute(dataMissingException);
-        }
+        errorMetricCallback.ifPresent(divErrorMetricCallback -> divErrorMetricCallback.execute(dataMissingException));
```

Comment:
Nitpick: this style of code instantiates a closure object, while the original form does not. Since DIV is used on the hot path, we do care about minimizing allocation, and so I think it's better to use a more traditional code style, even if slightly more verbose...

Reply:
This is a good point, will change it back. Thanks for reviewing this change, Felix!